src/Export.js

```diff
@@ -34,10 +37,65 @@ const Export = ({ API, year, session }) => {
       <SiMicrosoftexchange /> Office 365
     </Dropdown.Item>
     <Dropdown.Divider />
-    <Dropdown.Item eventKey="ics" download={`${name} as of ${new Date().toLocaleDateString().replace(/(\/|\\)/g,'.')}.ics`} href={`${window.location.protocol}//${path}`}>
+    <Dropdown.Item eventKey="ics" download={`${name} as of ${new Date().toLocaleDateString().replace(/(\/|\\)/g, '.')}.ics`} href={`${window.location.protocol}//${path}`}>
       <RiCalendar2Fill /> Download ICS file
     </Dropdown.Item>
+    <Dropdown.Item as="Button">
+      <div onClick={exportToPng}>
+        <BsCardImage /> Download PNG
```

Comment:
1. Perhaps this should say `Download PNG image`? Some people might not know/remember what a PNG is

2. A common confusion about the timetable is that different weeks may have different classes/times (the simplest example being a weekly tute starting in week 3). I think this should make it clear it's downloading the selected week's timetable as an image - maybe `Download PNG image for this week`? If you think that's too long / too much information don't worry about it

Reply:
Both good points, I've added 1) but maybe an alternate solution for 2). Do you think an FAQ would work until I can write a custom view as discussed in #84?